### PR TITLE
upload

### PR DESCRIPTION
**Pull request by papiyan98**
Triangle render

### CODE REVIEW

File: homework-3/index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Triangle Renderer</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <div class="container">
+    <input type="number" id="inputNumber" placeholder="Enter triangle length">
+    <button class="rotate-btn">Rotate</button>
+  </div>
+  <script type="module" src="./main.js"></script>
```
Comment (ao-alexi) [inline homework-3/index.html:14]: having a module script makes it impossible to run the code by opening the html file directly, requires a running server instead. if running your code has a prerequisite please include it in the Readme, or better yet just make it simple to run, that will make your colleagues' lives easier

File: homework-3/main.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import { renderTriangle, rotateTriangle } from "./triangleRenderer.js";
+
+const container = document.querySelector('.container');
+const rotateBtn = document.querySelector('.rotate-btn');
+const inputNumber = document.getElementById('inputNumber');
+
+let triangleElement, rotatedTriangleElement;
```
Comment (ao-alexi) [inline homework-3/main.js:7]: having non-constant global variables like these makes it easy to create a bug. for instance in this program that will happen if you press the rotate button several times before trying to render a new triangle
Screenshot 2023-07-10 at 03 02 24

**Review by ao-alexi — COMMENTED** [review]
good, there is input validation and the program works, please see the issues in the comments

File: homework-3/index.html (hunk 1 above)
Comment (papiyan98) [reply homework-3/index.html:14]: I've added this requirement in README.md, but technically it was possible to consistently include scripts in index.html and get rid of import/export.

**Pull request merged**